qiskit/scheduler/schedule_circuit.py

```diff
@@ -32,18 +32,22 @@ def schedule_circuit(circuit: QuantumCircuit,
     scheduled to occur as late as possible.
 
     Supported methods:
-        'as_soon_as_possible': Schedule pulses greedily, as early as possible on a qubit resource
-            alias: 'asap'
-        'as_late_as_possible': Schedule pulses late-- keep qubits in the ground state when possible
-            alias: 'alap'
+
+        * ``as_soon_as_possible``: Schedule pulses greedily, as early as possible on a qubit resource.
+          (alias: ``asap``)
+        * ``as_late_as_possible``: Schedule pulses late-- keep qubits in the ground state when possible.
```

Comment:
```suggestion
        * ``'as_late_as_possible'``: Schedule pulses late-- keep qubits in the ground state when possible.
```

Reply:
This line looks too long, is this why the build failed?

Reply:
yes, It should be fixed in [5c2d785](https://github.com/Qiskit/qiskit-terra/pull/3819/commits/5c2d78568cdf49a9d2be1f0b39f8500b0e2e61d3). Thank you. :)